.gitignore

```diff
@@ -1 +1,3 @@
 node_modules
+es
+lib
```

Comment:
They are not needed if there is only one file that already gets built because it's needed for testing, and so can be checked in just fine.